### PR TITLE
add single-node variant for applying cluster profile of single node

### PR DESCRIPTION
**Pull request by osherdp**
Compliments openshift/release#14556
We want to apply cluster profile of single-node, which enables 1 node that acts as control plane and compute resource

### CODE REVIEW

**Comment by osherdp** [comment]
not needed

File: prow.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ var supportedUpgradeTests = []string{"e2e-upgrade", "e2e-upgrade-all", "e2e-upgr
 var supportedPlatforms = []string{"aws", "gcp", "azure", "vsphere", "metal"}
 
 // supportedParameters are the allowed parameter keys that can be passed to jobs
-var supportedParameters = []string{"ovn", "proxy", "compact", "fips", "mirror", "shared-vpc", "large", "xlarge", "ipv6", "preserve-bootstrap", "test", "rt"}
+var supportedParameters = []string{"ovn", "proxy", "compact", "fips", "mirror", "shared-vpc", "large", "xlarge", "ipv6", "preserve-bootstrap", "test", "rt", "sno"}
```
Comment (smarterclayton) [inline prow.go:56]: `single-node`, same as the other PR

Comment (osherdp) [reply prow.go]: Done

**Comment by eranco74** [comment]
Allow deploying none HA cluster as described in openshift/enhancements#555

**Pull request merged**